src/tags/object/Video/HtxVideo.js

```diff
@@ -385,6 +369,28 @@ const HtxVideoView = ({ item }) => {
             disableView={!supportsRegions}
             framerate={item.framerate}
             controls={{ FramesControl: true }}
+            customControls={[
+              {
+                position: "left",
+                component: () => {
+                  return (
+                    <Dropdown.Trigger
+                      inline={isFullScreen}
+                      content={(
+                        <Menu size="medium" style={{ width: 150, minWidth: 0 }} closeDropdownOnItemClick={false}>
```

Comment:
anyway we can define a css rule to handle the width and min-width instead of inline styling it? these values are static